parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageReadStore.java

```diff
@@ -422,18 +417,15 @@ void addColumn(ColumnDescriptor path, ColumnChunkPageReader reader) {
     }
   }
 
-  void setBuffersToRelease(ByteBufferAllocator allocator, List<ByteBuffer> toRelease) {
-    this.allocator = allocator;
-    this.toRelease = toRelease;
+  void setReleaser(ByteBufferReleaser releaser) {
+    this.releaser = releaser;
```

Comment:
Should we check if the passed releaser is null?

Reply:
This is internal (both the method and the class are package private). I wouldn't do additional checks.